### DISCUSSION TITLE
Zigbee2MQTT devices do not show up in HA

### DISCUSSION THREAD

**Discussion by SirDarked** [Koenkk/zigbee2mqtt#20825] [General]
Hi everyone,

I tried out Zigbee2MQTT today with a second stick in addition to ZHA
integrated. Zigbee MQTT also works so far, my test sensor could also be connected and is also visible in the add-on. However, no device appears in HA. I suspect the problem is with the Mosquitto broker, which was already present in my system and is configured as a bridge for accessing SolarAssistant. In the integration of MQTT, only Solarassistant appears as a device with its entities.
In principle, I can also see the Z2MQTT topic in the MQTT explorer. It just doesn’t appear in HA. Unfortunately, I’m not really familiar with the MQTT story and hope to find some more expertise here.
Here is the configuration of Mosquitto for Solarassistant

![1](https://github.com/Koenkk/zigbee2mqtt/assets/109430610/e3597c9e-028d-48c7-a68a-2b9edfa41772)
![2](https://github.com/Koenkk/zigbee2mqtt/assets/109430610/7bddf7d4-dc77-48c9-b74d-0d626ab19aed)
![3](https://github.com/Koenkk/zigbee2mqtt/assets/109430610/951cab05-4064-4208-a851-9c9fb3703fe6)

Greetings
Carsten

**Comment by Koenkk** [comment]
You are probably using an outdated HA version, use at least 2023.8.0

**Comment by SirDarked** [comment]
Hi,
Thanks for your answer. Im using 2024.1.3
